tests/ut_common_test_file.py

```diff
@@ -0,0 +1,18 @@
+"""
+The unit test checks if build_component.py works fine.
```

Comment:
This actually tests the "build for testing" functionality. (We will want a test for "normal build" too.)